### ISSUE TITLE
Docker 2.3-rc3 crashes

### ISSUE THREAD

**Issue by dahankzter** [scylladb/scylla#3762]
*Installation details*
Scylla version (or git commit hash): 2.3-rc3
OS (RHEL/CentOS/Ubuntu/AWS AMI): Docker

Docker crash logs say:

```bash
➜  ~ docker run --name some-scylla scylladb/scylla:2.3-rc3            
running: (['/usr/lib/scylla/scylla_dev_mode_setup', '--developer-mode', '1'],)
/usr/lib/scylla/scylla_dev_mode_setup: line 15: verify_args: command not found
failed!
Traceback (most recent call last):
  File "/docker-entrypoint.py", line 13, in <module>
    setup.developerMode()
  File "/scyllasetup.py", line 28, in developerMode
    self._run(['/usr/lib/scylla/scylla_dev_mode_setup', '--developer-mode', self._developerMode])
  File "/scyllasetup.py", line 25, in _run
    subprocess.check_call(*args, **kwargs)
  File "/usr/lib64/python3.4/subprocess.py", line 558, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['/usr/lib/scylla/scylla_dev_mode_setup', '--developer-mode', '1']' returned non-zero exit status 127
```

**Comment by penberg** [comment]
I proposed the following fix earlier:

```
diff --git a/dist/docker/redhat/scylla-service.sh b/dist/docker/redhat/scylla-service.sh
index 6f4057ae7..e8aab730c 100755
--- a/dist/docker/redhat/scylla-service.sh
+++ b/dist/docker/redhat/scylla-service.sh
@@ -1,6 +1,8 @@
 #!/bin/bash

-. /usr/lib/scylla/scylla_prepare
+/usr/lib/scylla/scylla_prepare
+
+. /etc/sysconfig/scylla-server

 export SCYLLA_HOME SCYLLA_CONF
```

I will test it and send a patch.

**Comment by penberg** [comment]
The problem was already reported in #3647 and fixed in commit b52d647de2d2fbcce514934e2f9c62a00de7e6e1. However, the fix did not make it to 2.3-rc3 because `Dockerfile` was pointing to the wrong RPM repository. That is remedied by commit 590d1eb4cefa751ecd59a1df0c768b16036d6e58 as soon as it is promoted to `branch-2.3`.
